### DISCUSSION TITLE
href in NiceGUI

### DISCUSSION THREAD

**Discussion by afusdev** [zauberzeug/nicegui#3630] [Q&A] [answered]
### Question

Is there a way to make any element (specifically a ui.element) clickable like `<a href="#"></a>` in html? 
Would be a life saver not having to use a button every time something should be clickable.

Would be sick to see in a future update if it doesnt exist, like a `ui.element(href="#")`.
Thanks in advance! :)

**Comment by python-and-novella** [comment]
every element supports `click` envet listener and `href` props.
```python3
with ui.element('a').props('href="#"'):
    ui.html('link with href')
```

**Comment by falkoschindler** [comment, marked as answer]
There is also [`ui.link`](https://nicegui.io/documentation/link).
